documentation/src/main/asciidoc/stories/assembly_cache_modes.adoc

```diff
@@ -8,6 +8,25 @@
 * Optimizing databases by decreasing the volume of write operations.
 * Providing resiliency and durability for consistent data across clusters.
 
+[IMPORTANT]
+====
+{brandname} places a restriction of less than or equal to 255 characters on a cache name or a cache template name.
+
+If you exceed this character limit, {brandname} throws an exception.
+
+Try to write succinct cache names and cache template names.
+====
+
+[IMPORTANT]
+====
+A file system might set a limitation for the length of a file name, so ensure that a cache's name does not exceed this limitation.
```

Comment:
Not possible to define, because this can vary depending on what an admin sets for a filename. We need to be vague here, because its outside our control.